__tests__/app.test.js

```diff
@@ -34,4 +34,26 @@ describe('GET Requests', () => {
             });
         });
     });
+    test('/api/articles 200: responds with all articles & associated comment counts', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((res) => {
+            //Check Correct Length
+            expect(res.body.articles).toHaveLength(13);

```

Comment:
This is a good assertion, but a more flexible approach to this would be to use the .toBeGreaterThan() matcher, as this allows us to if needed expand the test database without breaking this test.